src/components/common/TmSessionWelcome.vue

```diff
@@ -14,7 +14,7 @@
         <LiSession
           v-if="!browserWithLedgerSupport"
           icon="usb"
-          title="Sign in with Ledger Nano S"
+          title="Sign in with Ledger Nano nbsp; S"
```

Comment:
This is rendered directly in Vue. You'll need to find another way to encode it.

Reply:
Ah OK my bad. Misunderstood. 

Were you thinking something more like this -> ` `